src/averageAges.js

```diff
@@ -14,12 +14,30 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
+const SEX_MALE = 'm';
+const SEX_FEMALE = 'f';
+
+function calculateMenAverageAge(people, century = null) {
```

Comment:
Do you really need to set the default value? You can simply check if this value exists.